src/components/scopedVariables/Table.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { TableItemI, TableListI } from './types'
+
+export const TableList = ({ children, width, headings }: TableListI) => {
+    return (
+        <div className={`scoped-variables-list-container`}>
```

Comment:
No need of interpolation can directly use css

Reply:
Done